app/main/views/questionnaire.py

```diff
@@ -240,33 +218,31 @@ def post_household_composition(eq_id, form_type, collection_id, group_id):
     questionnaire_manager = get_questionnaire_manager(g.schema, g.schema_json)
     answer_store = get_answer_store(current_user)
 
-    this_block = {
-        'block_id': 'household-composition',
-        'group_id': group_id,
-        'group_instance': 0,
-    }
+    this_location = Location(group_id, 0, 'household-composition')
```

Comment:
does lose some readability by not having the name arguments, what about (skipping obvious named group_id):

```python
this_location = Location(
    group_id, 
    group_instance=0,
    block_id='household-composition')
```

Just for the cases when the parameters being passed in aren't matching names to the function args.